client/coral-sign-in/containers/SignInContainer.js

```diff
@@ -119,8 +119,7 @@ class SignInContainer extends Component {
     }
   }
 
-  handleSignIn(e) {
-    e.preventDefault();
```

Comment:
so this line needs to stay in, since I was able to get the form params into the url accidentally since there's no `action` attribute on the form. I'm going to keep plugging away on a fix for this that doesn't involve removing this line.